packages/synapse-types/src/TrashedEntity.ts

```diff
@@ -3,11 +3,15 @@
  *
  * https://docs.synapse.org/rest/org/sagebionetworks/repo/model/TrashedEntity.html
  */
+import { EntityType } from './EntityType'
+
```

Comment:
Please move this import to the top of the file, so the above comment is applied to the `TrashedEntity` type

Reply:
Thanks @nickgros ! I just moved that line to the top